### PR TITLE
Add External Services Mock chart

### PR DESCRIPTION
**Pull request by tgorgol**
**Description**

Changes proposed in this pull request:

- Add External Services Mock chart

### CODE REVIEW

File: chart/compass/values.yaml [hunk 1]
```diff
@@ -257,7 +262,14 @@ global:
       maxOpenConnections: 1
       maxIdleConnections: 1
 
-  enableCompassDefaultScenarioAssignment: true
+  auditlog:
+    configmap:
```
Comment (pkosiec) [inline chart/compass/values.yaml:266]: `configMap`

File: chart/compass/values.yaml [hunk 2]
```diff
@@ -257,7 +262,14 @@ global:
       maxOpenConnections: 1
       maxIdleConnections: 1
 
-  enableCompassDefaultScenarioAssignment: true
+  auditlog:
```
Comment (pkosiec) [inline chart/compass/values.yaml:265]: `auditLog`?

File: chart/compass/charts/gateway/values.yaml [hunk 3]
```diff
@@ -10,9 +10,5 @@ gateway:
   enabled: true
   manageCerts: true
   auditlog:
-    enabled: false
-    authMode: "basic"
-    configmap:
-      name: "compass-gateway-auditlog-config"
-    secret:
-      name: "compass-gateway-auditlog-secret"
+    enabled: true # TODO: Switch to false after testing
```
Comment (pkosiec) [inline chart/compass/charts/gateway/values.yaml:13]: I think this value shouldn't be here anymore - we should use global value

File: chart/compass/charts/gateway/values.yaml [hunk 4]
```diff
@@ -10,9 +10,5 @@ gateway:
...
+    authMode: "oauth" # TODO: Switch to basic after testing
```
Comment (pkosiec) [inline chart/compass/charts/gateway/values.yaml:14]: I wonder if we shouldn't move it also to global, under `auditLog`

File: chart/compass/charts/external-services-mock/templates/auditlog-configmap.yaml [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+metadata:
+  name: {{ .Values.global.auditlog.configMapName }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ .Chart.Name }}
+    release: {{ .Release.Name }}
+data:
+  auditlog-oauth-user-var: "mock-user"
+  auditlog-oauth-tenant-var: "mock-tenant"
+  auditlog-url: "http://compass-external-services-mock.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.service.port }}"
```
Comment (dbadura) [inline chart/compass/charts/external-services-mock/templates/auditlog-configmap.yaml:12]: `auditlog-oauth-tenant-var` and `auditlog-oauth-user-var` is not needed.
We have default values, which are checked in `external-services-mock`.

File: chart/compass/charts/gateway/values.yaml (hunk 3 above)
Comment (pkosiec) [reply chart/compass/charts/gateway/values.yaml:14]: In future it would be good to move it to global values

**Pull request merged**